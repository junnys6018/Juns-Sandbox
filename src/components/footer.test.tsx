import { render } from '@testing-library/react';
import Footer from './footer';

jest.mock('./socials', () => {
    return {
        __esModule: true,
        SocialLinks: () => <div>Social Links Mock</div>,
    };
});

test('<Footer /> snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
});
