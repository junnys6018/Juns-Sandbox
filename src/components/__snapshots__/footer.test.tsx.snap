// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> snapshot 1`] = `
<DocumentFragment>
  <footer
    class="mt-auto text-center pb-4"
  >
    <div
      class="xl:hidden flex flex-row justify-center gap-4"
    >
      <div>
        Social Links Mock
      </div>
    </div>
    <span
      class="font-medium text-xs text-neutral-700"
    >
      Â© 2022 Designed & Built by Jun Lim 
    </span>
  </footer>
</DocumentFragment>
`;
